module/zstd/Makefile.in

```diff
@@ -16,13 +16,18 @@ ccflags-y += $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
 
 # Suppress unused but set variable warnings often due to ASSERTs
 ccflags-y += $(NO_UNUSED_BUT_SET_VARIABLE)
-# frame-larger-than=20480 is needed here due to requirements of the stock zstd
-ccflags-y += -Wframe-larger-than=20480 -O3
 
-$(MODULE)-objs += zstd.o
+# Quiet warnings about frame size due to use of an unmodified zstd lib
```

Comment:
Maybe turn this into:
`# Quiet warnings about frame size due to unused code in unmodified zstd lib`

Reply:
good point